pkg/ocm/client.go

```diff
@@ -25,17 +27,52 @@ type Config struct {
 	Debug        bool   `envconfig:"OCM_DEBUG" default:"false"`
 }
 
+type SdKLogger struct {
+	Log         *logrus.Logger
+	FieldLogger logrus.FieldLogger
+}
+
+func (l *SdKLogger) DebugEnabled() bool {
+	return l.Log.IsLevelEnabled(logrus.DebugLevel)
+}
+
+func (l *SdKLogger) InfoEnabled() bool {
+	return l.Log.IsLevelEnabled(logrus.InfoLevel)
+}
+
+func (l *SdKLogger) WarnEnabled() bool {
+	return l.Log.IsLevelEnabled(logrus.WarnLevel)
+}
+
+func (l *SdKLogger) ErrorEnabled() bool {
+	return l.Log.IsLevelEnabled(logrus.ErrorLevel)
+}
+
+func (l *SdKLogger) Debug(ctx context.Context, format string, args ...interface{}) {
+	l.FieldLogger.Debugf(format, args...)
+}
+
+func (l *SdKLogger) Info(ctx context.Context, format string, args ...interface{}) {
+	l.FieldLogger.Infof(format, args...)
+}
+
+func (l *SdKLogger) Warn(ctx context.Context, format string, args ...interface{}) {
+	l.FieldLogger.Warnf(format, args...)
+}
+
+func (l *SdKLogger) Error(ctx context.Context, format string, args ...interface{}) {
+	l.FieldLogger.Errorf(format, args...)
+}
+
 func NewClient(config Config) (*Client, error) {
-	// Create a logger that has the debug level enabled:
-	logger, err := sdkClient.NewGoLoggerBuilder().
-		Debug(config.Debug).
-		Build()
-	if err != nil {
-		return nil, fmt.Errorf("Unable to build OCM logger: %s", err.Error())
+	log := logrus.New()
+	if config.Debug {
```

Comment:
I was thinking that we want to have different logger so that we can have DEBUG for the service not enabling DEBUG for OCM SDK which can be very verbose.
Thoughts?

Reply:
@ronniel1 ping

Reply:
@filanov what do you think?

Reply:
I think that it's going to be confusing to have multiple loggers.
When we create the logger in the main we can add parameters or wrappers there. if we create it inside the code we have less control and now we will need to support specific configuration for this logger. 
So i prefer to create the logger in the main as @ronniel1 suggested.

Reply:
Done